### DISCUSSION TITLE
Compose Profiles

### DISCUSSION THREAD

**Discussion by dimaguy** [louislam/dockge#375] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The compose standard supports bundling select containers of a compose file in a so called profile, which allows to launch the same software solution in different shapes, it'd be nice to be able to start and see profiles of a compose file from Dockge.

More info: https://docs.docker.com/compose/profiles/

### ✔️ Solution

I believe a good solution would be adding a dropdown menu to the start button (maybe a small section of the button with a down arrow) that lists the profiles in order to launch them

**Comment by satr14washere** [comment]
bump
